cross/rar2fs/Makefile

```diff
@@ -0,0 +1,27 @@
+PKG_NAME = rar2fs
+PKG_VERS = 1.29.0
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+2PKG_DIST_SITE = https://github.com/hasse69/rar2fs/releases/download/v$(PKG_VERS)/
```

Comment:
looks like a typo
PKG_DIST_SITE (not 2PKG_DIST_SITE)

Reply:
You're absolutely correct. A change has been pushed.